default.json5

```diff
@@ -119,6 +95,7 @@
       matchUpdateTypes: [
         "minor",
       ],
+      matchCurrentVersion: "!/^0/",
```

Comment:
```suggestion
```
We won't need this because there is an overriding rule down below.

Reply:
the rules are merged, not overridden, as we saw in the test